projects/helper/coreAssets.json

```diff
@@ -128,7 +128,10 @@
     "IVN": "0x6a46d878401f46b4c7f665f065e0667580e031ec",
     "ankrBNB": "0xe85afccdafbe7f2b096f268e31cce3da8da2990a",
     "aBNBb": "0xbb1aa6e59e5163d8722a122cd66eba614b59df0d",
-    "TUSD": "0x40af3827f39d0eacbf4a168f8d4ee67c121d11c9"
+    "TUSD": "0x40af3827f39d0eacbf4a168f8d4ee67c121d11c9",
+    "wstETH": "0x26c5e01524d2E6280A48F2c50fF6De7e52E9611C",
+    "BBTC": "0xF5e11df1ebCf78b6b6D26E04FF19cD786a1e81dC",
```

Comment:
also, can you remove BBTC & solvBTC from here and declare it directly in your file? the tokens here are sort of whitelisted and used for pricing other tokens